.github/workflows/rspec_tests.yml

```diff
@@ -11,13 +11,10 @@ jobs:
     strategy:
       matrix:
         cfg:
-          - {os: ubuntu-latest, ruby: 2.4}
-          - {os: ubuntu-latest, ruby: 2.5}
           - {os: ubuntu-latest, ruby: 2.6}
           - {os: ubuntu-latest, ruby: 2.7}
           - {os: ubuntu-latest, ruby: 3.2}
-          - {os: ubuntu-latest, ruby: jruby-9.2.10.0}
-          - {os: windows-2019, ruby: 2.5}
+          - {os: ubuntu-latest, ruby: jruby-9.3}
```

Comment:
jruby 9.4 is out now and ships in Puppet Server 8 along with Ruby 3.2. It'd be great, if we're going to drop support for older versions we added support for newer ones. I'd love to see Ruby 3.1 & 3.2 in addition to JRuby 9.4 added to this matrix.